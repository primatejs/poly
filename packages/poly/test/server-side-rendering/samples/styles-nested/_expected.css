div.poly-bzh57p{color:red}
div.poly-4yw8vx{color:green}