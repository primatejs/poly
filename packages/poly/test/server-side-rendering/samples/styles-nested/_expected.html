<div class="poly-bzh57p">red</div>
<div class="poly-4yw8vx">green: foo</div>



<div class="poly-4yw8vx">green: bar</div>


